packages/frontend/src/components/ProjectConnection/DbtForms/AzureDevOpsForm.tsx

```diff
@@ -80,6 +81,8 @@ const AzureDevOpsForm: FC<{ disabled: boolean }> = ({ disabled }) => {
                 })}
                 disabled={disabled}
             />
+            <DbtVersionSelect disabled={disabled} />
```

Comment:
we want this dbt version after `repo`, so I extracted this from the common  form, and added to all forms (except CLI)